docs/0-installation.md

```diff
@@ -112,6 +112,30 @@ Draper::CollectionDecorator.send :delegate, :per_page_kaminari
 
 If you're getting the error `wrong number of arguments (6 for 4..5)`, [read #2703].
 
+## webpacker
+
+Since rails 6.0, webpacker has become the default asset generator. You can opt into using webpacker for ActiveAdmin assets as well by setting `use_webpacker` at installation time or manually.
```

Comment:
Just a slight difference since its technically new apps built with Rails 6 that get the default but not if you upgrade. I think this helps be a little clearer as to why its opt in for AA and would help avoid confusion.

```suggestion
For new apps starting with Rails 6.0, Webpacker has become the default asset generator. You can **opt-in to using Webpacker for ActiveAdmin assets** as well by updating your configuration to turn on the `use_webpacker` option, either at installation time or manually.
```